src/cmd/src/bin/greptime.rs

```diff
@@ -46,6 +46,8 @@ enum SubCommand {
     Metasrv(metasrv::Command),
     #[clap(name = "standalone")]
     Standalone(standalone::Command),
+    #[clap(name = "version")]
+    Version(version::Command),
```

Comment:
I prefer to treat this as an option `--version` (and `-v` for short). Subcommand is for components

Reply:
![image](https://user-images.githubusercontent.com/9639449/204071936-7bdc73dc-67cb-45a3-a67d-36dae655c4a0.png)

`--version` is reserved by clap and it uses `-V` for short, simply overwrite the default action.